game/script-greetings.rpy

```diff
@@ -972,6 +972,79 @@ label greeting_love_plus_show_yourself:
 
     return
 
+init 5 python:
+    registerTopic(
+        Topic(
+            persistent._greeting_database,
+            label="greeting_love_plus_amazing_scenery",
+            unlocked=True,
+            affinity_range=(jn_affinity.LOVE, None),
+            additional_properties={
+                "expression": "2cslbol"
+            }
+        ),
+        topic_group=TOPIC_TYPE_GREETING
+    )
+
+label greeting_love_plus_amazing_scenery:
+    n 4unmbgleex "Aha!{w=0.75}{nw}"
+    extend 3fchbgl " [player]!{w=0.75}{nw}"
+    extend 6fcssmlsbl " I-{w=0.2}I knew you'd show up eventually!"
+    n 1fslsslsbl "Heh.{w=0.75}{nw}"
+    extend 4fllbgl " After all..."
+    n 2fcsbgledz "I don't see any other pieces of {w=0.2}{i}amazing{/i}{w=0.2} scenery around here!"
+    n 2fsqsml "Ehehe."
+    $ chosen_tease = jn_utils.getRandomTease()
+    n 2fchblleaf "Welcome back,{w=0.2} [chosen_tease]!"
+
+    return
+
+init 5 python:
+    registerTopic(
+        Topic(
+            persistent._greeting_database,
+            label="greeting_love_plus_manga_chapters",
+            unlocked=True,
+            conditional="jn_desk_items.getDeskItem('jn_parfait_manga_held').unlocked",
+            affinity_range=(jn_affinity.LOVE, None),
+            additional_properties={
+                "desk_item": "jn_parfait_manga_held",
+                "expression": "1cdwca"
+            }
+        ),
+        topic_group=TOPIC_TYPE_GREETING
+    )
+
+label greeting_love_plus_manga_chapters:
+    n 1cdwpu "..."
+    n 1tdwsl "..."
+    n 1tnmboeqm "...?{w=0.75}{nw}"
+    n 1unmgsleshsbr "Ah!{w=0.75}{nw}"
+    $ player_initial = jn_utils.getPlayerInitial()
+    extend 1cllbglsbr " [player_initial]-{w=0.2}[player]!{w=0.75}{nw}"
+    extend 1cchbglsbr " What's up?"
+    n 1fchsmlsbr "..."
+    n 1cnmpul "Huh?{w=0.75}{nw}"
+    extend 1udwaj " Oh.{w=0.75}{nw}"
+    extend 1ulrbo " Don't mind the manga."
+    n 1fcsbglsbl "{i}Our{/i} chapters together are way more interesting,{w=0.2} a-{w=0.2}anyway.{w=0.75}{nw}"
+    extend 1fchsmlsbl " Ehehe."
+
+    show black zorder JN_BLACK_ZORDER with Dissolve(0.5)
+    $ jnPause(0.5)
+    show natsuki 3nlrsmleme
+    $ Natsuki.clearDesk()
+    play audio drawer
+    $ jnPause(1.3)
+    hide black with Dissolve(0.5)
+    $ jnPause(1)
```

Comment:
Imo it would be nice to add an extra prop which will let the manga "stay" at the desktop like our favourite plant Sanjo.

Reply:
I wouldn't mind adding that if we have the asset for it.

Reply:
I tried to add it, please take a look when time allows.